Disable opacity for font colors

saihaj opened the pull request:
### Summary of PR
Add `disableAlpha` prop type to turn off alpha for color picker

Alpha slider is not shown for the following options in "Text":
* Primary and Secondary 
  * Font Color
  * Larivaar Assist Color
  * Drop Color

**Before**
<img width="236" alt="image" src="https://user-images.githubusercontent.com/44710980/83082144-307fe900-a048-11ea-9d6c-400143bcde12.png">

**After**
<img width="337" alt="image" src="https://user-images.githubusercontent.com/44710980/83082041-e139b880-a047-11ea-9ab2-ebe1d4c17c60.png">


### Tests for unexpected behavior
Colors can be changed with no issues.

### Time spent on PR
15 minutes

### Linked issues
Fix #26 

### Reviewers
@Harjot1Singh @bhajneet

saihaj committed:
fix: disable opacity for font colors

Fixes #26

harjot1singh commented on src/components/SettingComponent.js:
```diff
@@ -95,7 +95,7 @@ Button.defaultProps = {
 }
 
-export const PopoverColorPicker = ( { name, value, icon, storageKey, onChange, ...props } ) => {
+export const PopoverColorPicker = ( { name, value, icon, storageKey, onChange, disableAlpha, ...props } ) => {
```
You don't actually need to specify the `disableAlpha` prop here if you're just passing it through as-is in L140, since `{...props}` will spread it for you.

saihaj committed:
refactor: remove specified prop

as mentioned in:
https://github.com/ShabadOS/theme-tool/pull/28#discussion_r431516595
